docs/sources/development/contributing.md

```diff
@@ -72,11 +72,11 @@ If your pull request is not accepted on the first try, don't be discouraged! If
 
 We're trying very hard to keep Eclipse SmartHome lean and focused. We don't want it to do everything for everybody. This means that we might decide against incorporating a new feature. However, there might be a way to implement that feature on top of Eclipse SmartHome.
 
-## Discuss your Design on the mailing list
+## Discuss your design in the discussion forum
```

Comment:
Well, there are also other examples like https://github.com/docker/docker/blob/master/CONTRIBUTING.md
I only adapted these headers to comply with the header that is in the same file just above :-)

btw, if you use capitalized headers, it is not only for nouns, but for all words that are not "filling" words.